qa.md

```diff
@@ -1,5 +1,13 @@
 # Q&A list managed by moderators
 
-## Q. example
+## Q. When distribution?
 
-A. example
+A. TBA. It will be after mainnet launch in May 2022.
+
+## Q. How much $GUU can I get from Airdrop campaign?
+
+A. It's not decided yet. We will announce before the distribution. Keep your eyes on our Discord(https://discord.gg/ksPWKDjH4s) and Twitter(https://twitter.com/ununifi).
+
+## Q. Can I change my address which I will receive airdrop?
+
+A. Yes. We are planning to get rid of scammer address and correct the address of the participants after mainnnet launch.
```

Comment:
```suggestion
A. Yes. We are planning to get rid of scammer address and correct the address of the participants after mainnet launch.
```

Reply:
## Q. When is the death line that I can inform in "Salvation for people who are not in the final list even they submitted the airdrop campaign form" ?
- 9th May 23:59 UTC.